docs/develop/01_sdk/01_quickstart.md

```diff
@@ -133,34 +136,49 @@ We will move onto connecting to the **KILT blockchain**.
 
 ### Connect to the KILT Blockchain
 
-Connecting to and disconnecting from the KILT blockchain is required for any operation that relies on the KILT blockchain, such as **querying and verifying a credential**.
-
-Still within the same `main` function, you need to configure the SDK to connect to a **KILT node**.
-For this, the SDK exposes **`Kilt.connect()`** to configure the address of the node to connect to.
+To perform operations that rely on the KILT blockchain, such as querying and verifying a credential, it's essential to connect to the KILT blockchain.
+Within the same `main` function, configure the SDK to connect to a KILT node using the `Kilt.connect()` method:
+
+<Tabs groupId="chain-choice">
+  <TabItem value='pere' label='Peregrine (Testnet)' default>
+    Peregrine is the development blockchain.
+    For testing and development purposes connect to this network.
+    <SnippetBlock
+      className="language-ts"
+      funcEnd="return"
+      >
+      {ConnectPere}
+    </SnippetBlock>
+  </TabItem>
+  <TabItem value='spirit' label='Spiritnet (Production)'>
+    Spiritnet is the production blockchain.
+    When you are ready to publish your DApp, connect to the Spiritnet network for production purposes.
+    <SnippetBlock
+      className="language-ts"
+      funcEnd="return"
+      >
+      {ConnectSpirit}
+    </SnippetBlock>
+  </TabItem>
+</Tabs>
 
-We will use the official **Spiritnet** address:
+To ensure proper cleanup, call the `Kilt.disconnect()` function at the bottom of the main function:
 
 <SnippetBlock
   className="language-ts"
-  funcEnd="return"
 >
-  {Connect}
+  {Disconnect}
 </SnippetBlock>
 
-After establishing a connection, you have access to the chain, but let's not forget to **close** any connections when we are done!
-Connections to blockchain nodes should be dropped when no longer needed: to do that simply call the `Kilt.disconnect()` function at the bottom of `main` function.
+By adding `await Kilt.disconnect()` at the end of the main function, you ensure that the connection to the blockchain node is properly closed when the script finishes executing.
+This helps maintain the integrity of your application and is a good practice to follow.
```

Comment:
What does integrity means?

Reply:
keeping the app state free from resource leaks which could lead to a crash. [Integrity as in an unimpaired condition](https://www.merriam-webster.com/dictionary/integrity)